front/views/settings/PrivacyView.tsx

```diff
@@ -1,31 +1,52 @@
 import React from "react";
-import { View } from 'react-native';
+import { View } from "react-native";
 import { Center, Button, Text, Switch, Heading } from "native-base";
 import { translate } from "../../i18n/i18n";
+import ElementList from "../../components/GtkUI/ElementList";
 
-const PrivacyView = ({navigation}) => {
-    return (
-        <Center style={{ flex: 1}}>
-            <Heading style={{ textAlign: "center" }}>{ translate('privBtn')}</Heading>
+const PrivacyView = ({ navigation }) => {
+	const [dataCollection, setDataCollection] = React.useState(false);
```

Comment:
Shouldn't this be selected  from the store?

Reply:
Yes I think so but I didn't looked into the store previously the switch wasn't linked to anything